demo/demo.js

```diff
@@ -141,11 +141,12 @@ ethSignMsg.addEventListener("click", async () => {
 
 // Display single-sig BTC address on device for verification
 document.getElementById("btcAddressSimple").addEventListener("click", async () => {
-    await device.api.btcDisplayAddressSimple(
+    const address = await device.api.btcDisplayAddressSimple(
         constants.messages.BTCCoin.BTC,
         getKeypathFromString("m/49'/0'/0'/0/0"),
         constants.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
     );
+    alert("btcDisplayAddressSimple returned:\n" + address);
```

Comment:
What you could do for a more natural workflow:

- get address without displaying (display param = false)
- show  the address
- display address on BitBox02

Reply:
PTAL https://github.com/digitalbitbox/bitbox02-api-js/pull/40/commits/f4f2d0d9fa2a4b92a9b2aeeb9c6e7805887f26dd